src/components/bookmarks/addBookmark/addBookmark.css

```diff
@@ -0,0 +1,87 @@
+@import '../../app/variablesV2.css';
+@import '../../app/mixins.css';
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+
+  & .content {
+    width: 720px;
+
+    & header {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+    }
+
+    & h1 {
+      @mixin headingLarge;
+    }
+  }
+}
+
+.formHolder {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+
+  & > .buttonHolder {
+    display: flex;
+    margin: 16px auto 30px;
+    width: 390px;
+
+    & button {
+      display: block;
+      width: 100%;
+    }
+  }
+
+  & .fieldGroup {
+    display: block;
+    position: relative;
+
+    & .avatar {
+      left: 16px;
+      position: absolute;
+      top: 12px;
+    }
+  }
+
+  & label {
+    display: block;
+    margin-bottom: 24px;
+
+    & > .label {
+      @mixin contentNormal bold;
+
+      color: var(--color-maastricht-blue);
+      display: block;
+      margin-bottom: 12px;
+    }
+
+    & .input[name="address"] {
+      padding-left: 48px;
+    }
+
+    & .input:focus ~ .feedback {
+      opacity: 1 !important;
+    }
+  }
+
+  & .feedback {
```

Comment:
Can at least some of these styles be inside the `Feedback` component?

Reply:
Maybe it's better to create a ticket to update places using the feedback component, since it should be consistent across all the application, and I'm worried of adding the rules to the component and breaking the component  in another place.

Reply:
ok